website/content/docs/nia/usage/zscaler-application-segments.mdx

```diff
@@ -0,0 +1,404 @@
+
+![NIA CTS with ZPA application segments workflow](/img/nia/zpa-application-segments-diagram-workflow.png)
+
+Complete the following steps to configure your ZPA application segment:
+
+1. Register your Consul services
+1. Configure CTS
+1. Start CTS
+1. Verify NIA workflow
+
+## Background
+
+CTS is a tool that uses Consul's service catalog as a source of truth for all applications running in a given environment. When it detects changes to the services (for example, a new service node is added or deleted from the service list), CTS dynamically applies the necessary changes to your infrastructure using Terraform, in this case, the ZPA application segment resources.
+
+ZPA [application segments](https://help.zscaler.com/zpa/about-applications) are groups of applications that often share access type or user privileges. You can use ZPA application segments to reduce your application's attack surface by limiting access to extraneous ports and specific users.
+
+The [`application-segment-nia`](https://registry.terraform.io/modules/zscaler/application-segment-nia/zpa/latest) Terraform module, a ZScaler-managed Terraform module for CTS, lets operators automate their network infrastructure by dynamically configuring domain entries in your application segments based on changes to the respective services in Consul.
+
+The following diagram shows an example network infrastructure automation (NIA) workflow with Terraform, Consul, CTS, and ZPA.
+
+1. Consul catalog updates with new service registrations or changes
+1. CTS subscribes to updates from Consul catalog
+1. When CTS detects changes from Consul catalog, it initiates run with the specified module to apply the changes
+
+![NIA CTS with ZPA high-level workflow diagram](/img/nia/cts-zpa-workflow.png)
+
+<Tip>
+
+ZScaler also manages the [`application-server-nia`](https://registry.terraform.io/modules/zscaler/application-server-nia/zpa/latest) Terraform NIA module. This module will create and update application servers in ZPA when CTS detects changes in Consul services.
+
+</Tip>
+
+This topic assumes you are familiar with the standard Consul workflow and CTS. If you are unfamiliar with these topics, reference their respective tutorials to learn more.
+
+- Complete the [Get Started with Consul on VMs](/consul/tutorials/get-started-vms) or [Get Started with Consul on Kubernetes](/consul/tutorials/get-started-kubernetes) tutorials to learn more about Consul.
+- Complete the [Network Infrastructure Automation with Consul-Terraform-Sync Intro](/consul/tutorials/network-infrastructure-automation/consul-terraform-sync-intro) tutorial to learn more about CTS.
+
+## Requirements
+
+1. A Zscaler Private Access (ZPA) Tenant
+1. ZPA API Credentials. You will use these credentials to authenticate both Terraform and CTS to create and manage resources in ZPA Cloud.
+1. A [Consul datacenter](#consul) for service networking and service mesh
```

Comment:
The instructions listed as well as the service configuration files seem to be Service Discovery only. 

This line could be misleading.